clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -2537,6 +2537,47 @@ static RValue EmitHipStdParUnsupportedBuiltin(CodeGenFunction *CGF,
   return RValue::get(CGF->Builder.CreateCall(UBF, Args));
 }
 
+static void buildInstrinsicCallArgs(CodeGenFunction &CGF, const CallExpr *E,
```

Comment:
Shouldn't need any CGBUiltin changes?

Reply:
Yeah, no need of anything change in CGBuiltin.